examples/llama-2/qlora.yml

```diff
@@ -6,6 +6,8 @@ is_llama_derived_model: true
 load_in_8bit: false
 load_in_4bit: true
 strict: false
+gpu_memory_limit: 20GiB
+lora_on_cpu: true
```

Comment:
I don't think adding these for default yaml would be good as a user might not need this and build off this yaml unknowingly.
```suggestion
```

Reply:
Removed.